app/application.py

```diff
@@ -72,19 +89,12 @@ def logged_in():
             # userID = decodedJWT['username']
             # userName = userID.split('@')[0]
 
-            # Build the survey data (To Do survey type)
-            dataArray = build_survey_data()
-
-            # Filter the data array to remove surveys that shouldn't appear on the To Do page
-            allowedStatuses = ['Not started', 'In progress']
-
-            # TODO - the line below can be commented out to demonstrate sorting. This comment can be removed eventually.
-            dataArray = filter_surveys(dataArray, allowedStatuses)
+            # Filters the data array to remove surveys that shouldn't appear on the To Do page
+            statusFilter = {'status_filter': '["Not started", "In progress"]'}
```

Comment:
You have a list inside a string????? I'm not sure why you are doing that? Try this, in your terminal type python then paste this code into your >>>.
>>> mydict = {'status':['not started','started','maybe']}
>>> mydict['status']
['not started', 'started', 'maybe']
>>> for key in mydict['status']:
...     print key
... 
not started
started
maybe
>>>

You pass the string in the header, so maybe the other side de-seriealises it and jobs a good'n but from a perfect python sense it's a bit strange! Are you happy with this implementation ?